platformio.ini

```diff
@@ -98,3 +98,11 @@ framework = arduino
 board = sanguino_atmega644p
 build_flags = ${common.build_flags}
 lib_deps = ${common.lib_deps}
+
+[env:tronxyx5s]
+platform = atmelavr
+framework = arduino
+board = sanguino_atmega1284p
+build_flags = ${common.build_flags}
+upload_speed = 57600
+lib_deps = ${common.lib_deps}
```

Comment:
Since this is identical to anet10 I'm getting the sense that they might both be combined as `env:melzi`.